packages/relay/src/lib/eth.ts

```diff
@@ -646,23 +647,24 @@ export class EthImpl implements Eth {
   async sendRawTransaction(transaction: string): Promise<string | JsonRpcError> {
     this.logger.trace('sendRawTransaction(transaction=%s)', transaction);
 
+    const parsedTx = ethers.utils.parseTransaction(transaction);
     try {
-      await this.precheck.gasLimit(transaction);
-      await this.precheck.nonce(transaction);
-      this.precheck.chainId(transaction);
-      this.precheck.value(transaction);
+      this.precheck.gasLimit(parsedTx);
```

Comment:
Based on the above this would simplify to 
```suggestion
      this.precheck.sendRawTransactionCheck(transaction);
```